paimon-core/src/main/java/org/apache/paimon/table/source/DataSplit.java

```diff
@@ -316,6 +317,9 @@ private static FunctionWithIOException<DataInputView, DataFileMeta> getFileMetaS
             DataFileMeta08Serializer serializer = new DataFileMeta08Serializer();
             return serializer::deserialize;
         } else if (version == 2) {
+            DataFileMeta09Serializer serializer = new DataFileMeta09Serializer();
+            return serializer::deserialize;
+        } else if (version == 3) {
```

Comment:
public static final int DATA_FILE_META_VERSION_1 = 1;
public static final int DATA_FILE_META_VERSION_2= 2;
public static final int DATA_FILE_META_VERSION_3= 3;

Reply:
No need to do this.

Reply:
OK